src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -283,7 +287,6 @@ private void DownloadPackage(object sender, RoutedEventArgs e)
         internal void DownloadSpecifiedPackageAndRefresh(PackageDependencyInfo info)
         {
             packageInstaller.DownloadAndInstallPackage(info);
-            DependencyRegen(currentWorkspace);
```

Comment:
This is not needed as the DependencyRegen is being called after the package is loaded